concepts/60 Themes and Styles/05 Predefined Themes/50 Apply a Theme/05 Using SCSS.md

```diff
@@ -0,0 +1,47 @@
+You can import the theme bundle from the `bundles` folder from the `devextreme` package in the root `.scss` file:
+
+    /* @use 'devextreme/scss/bundles/dx.theme_name.scss';  */
+    @use 'devextreme/scss/bundles/dx.carmine.scss';
+
+Or you can import individual files for the whole theme. The example below imports files of the `Material Blue Light Compact` theme
+
+    @use 'devextreme/scss/widgets/material/colors' with ($color: 'blue', $mode: 'light');
+    @use 'devextreme/scss/widgets/material/sizes' with ($size: 'compact');
+    @use 'devextreme/scss/widgets/material/icons';
+    @use 'devextreme/scss/widgets/material/widget';
+    @use 'devextreme/scss/widgets/material/card';
+    @use 'devextreme/scss/widgets/material/fieldset';
+    @use 'devextreme/scss/widgets/material/common';
+    @use 'devextreme/scss/widgets/base/resizable';
+    @use 'devextreme/scss/widgets/base/draggable';
+    @use 'devextreme/scss/widgets/base/ui';
+    @use 'devextreme/scss/widgets/material';
+
+The theme path can be either `material` or `generic` depending on the theme family. The second solution is useful if you need to import only specific components. To do this, replace `@use 'devextreme/scss/widgets/material';` with `@use 'devextreme/scss/widgets/componentName';`:
+
+    @use "devextreme/scss/widgets/generic/colors" with ($color: "darkmoon");
+    @use "devextreme/scss/widgets/generic/sizes" with ($size: "default");
+    @use "devextreme/scss/widgets/generic/typography";
+    @use "devextreme/scss/widgets/generic/icons";
+    @use "devextreme/scss/widgets/generic/widget";
+    @use "devextreme/scss/widgets/generic/card";
+    @use "devextreme/scss/widgets/generic/fieldset";
+    @use "devextreme/scss/widgets/generic/common";
+    @use "devextreme/scss/widgets/base/resizable";
+    @use "devextreme/scss/widgets/base/draggable";
+    @use "devextreme/scss/widgets/base/ui";
+    @use "devextreme/scss/widgets/generic/textBox";
+    @use "devextreme/scss/widgets/generic/button";
+
+You can use the following variables in the imports:
+
+- `$mode`    
+The theme main palette (`dark` or `light`). This variable has effect for `Material` themes only.
+
+- `$color`    
+Derived from the theme name. Represents the theme's main color (blue, carmine, contrast, etc.)
+
+- `$size:`    
+Can be `compact` or `default`.
+
+[note] If you want to customize inner variables (like components fonts, color, etc.), examine the file content inside the `devextreme/scss/widgets/` folder to learn what variables can be customized. Note that this code is internal and DevExtreme does not have a complete documentation for the SCSS structure. Variable names, order of import, and the folder structure may be changed without any warning. To customize specific variables, use [ThemeBuilder](/Documentation/Guide/Themes_and_Styles/ThemeBuilder/) or override [individual CSS classes](/Documentation/Guide/Themes_and_Styles/HTML-Based_Components_Customization/#Individual_CSS_Classes).
```

Comment:
> like component**s** fonts

Agreement of numbers. You wanted to say "fonts of components". However, a more compact form would look only as follows:

> component font**s**

Only the last word is a noun, and only this noun can be plural.

> examine **the** file content inside

The word "content" is uncountable. Is this case definite? No. We don't know anything of this content in this context, and it's entirely unimportant.

> **a** complete documentation

Like "text", documentation is uncountable. While it's possible to interpret "documentation" as a piece of documentation - a documentation repository, a documentation topic, etc. - this is too far-fetched in this context.

```suggestion
[note] If you want to customize internal variables (like component fonts, color, etc.), examine file content inside the `devextreme/scss/widgets/` folder to learn about customizable variables. Note that this code is internal and DevExtreme does not have complete documentation for the SCSS structure. Variable names, import order, and the folder structure may change in future builds without official notice. To customize specific variables, use [ThemeBuilder](/Documentation/Guide/Themes_and_Styles/ThemeBuilder/) or override [individual CSS classes](/Documentation/Guide/Themes_and_Styles/HTML-Based_Components_Customization/#Individual_CSS_Classes).
```